Ginger/GingerCoreNET/Application Models/Delta/PomDelta/PomDeltaUtils.cs

```diff
@@ -509,6 +509,8 @@
 
         public void UpdateOriginalPom()
         {
+            MapDeletedElementWithNewAddedElement();
```

Comment:
can we handle it in the deleted element if condition itself instead of a new function ? Because new function is iterating over all element again